library/Makefile

```diff
@@ -91,9 +104,7 @@ shared: libmbedcrypto.$(DLEXT) libmbedx509.$(DLEXT) libmbedtls.$(DLEXT)
 # tls
 libmbedtls.a: $(OBJS_TLS)
 	echo "  AR    $@"
-	$(AR) -rc $@ $(OBJS_TLS)
-	echo "  RL    $@"
-	$(AR) -s $@
+	$(AR) $@ $(OBJS_TLS)
```

Comment:
Regarding AR I think you mean ```iarchive```. ```--symbols``` dumps the symbols in a ```.a``` file. ```ar``` does not have anything equivalent to this. Closest is ```nm``` command. But that does not relate here.
```ar``` help shows this for ```-s``` option:
```[s]          - create an archive index (cf. ranlib)```